cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/Transactions.java

```diff
@@ -66,6 +70,52 @@ public static TransactionExecutor createTransactionExecutor(TransactionExecutorF
     return factory.createExecutor(ImmutableList.of(txAware));
   }
 
+  /**
```

Comment:
this essentially removes the need for TransactionExecutor? Also I think this class could be moved to Tephra.